### PR TITLE
[T] (quests) basic_create

### PR DESCRIPTION
**Pull request by luo-2233**
Co-authored-by: 海福 <18917570841@163.com>

### CODE REVIEW

**Review by Cactusstudent — APPROVED** [review]

File: config/ftbquests/quests/chapters/basic_create.snbt [hunk 1]
```diff
@@ -77,7 +77,7 @@
 		{
 			x: 1.5d
 			y: 4.5d
-			description: ["Uses rotational power to crush items down"]
+			description: ["使用旋转能粉碎物品。"]
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/basic_create.snbt]:
```suggestion
			description: ["使用旋转力粉碎物品"]
```

File: config/ftbquests/quests/chapters/basic_create.snbt [hunk 2]
```diff
@@ -31,7 +31,7 @@
 		{
 			x: 1.5d
 			y: 1.5d
-			description: ["Moves rotational energy"]
+			description: ["转移旋转能"]
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/basic_create.snbt]:
```suggestion
			description: ["转移旋转力"]
```

File: config/ftbquests/quests/chapters/basic_create.snbt [hunk 3]
```diff
@@ -60,7 +60,7 @@
 		{
 			x: 1.5d
 			y: 0.0d
-			description: ["Manual rotational energy"]
+			description: ["人工产生旋转能"]
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/basic_create.snbt]:
```suggestion
			description: ["人工产生旋转力"]
```

File: config/ftbquests/quests/chapters/basic_create.snbt [hunk 4]
```diff
@@ -111,7 +111,7 @@
 		{
 			x: 1.5d
 			y: 6.0d
-			description: ["Comes from milling gravel"]
+			description: ["通过研磨砂砾获取。"]
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/basic_create.snbt]:
```suggestion
			description: ["通过研磨砂砾获取"]
```

File: config/ftbquests/quests/chapters/basic_create.snbt [hunk 5]
```diff
@@ -237,7 +237,7 @@
 		{
 			x: -1.5d
 			y: 1.5d
-			description: ["Allows you to find certain biomes"]
+			description: ["能让你找到特定的生物群系。"]
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/basic_create.snbt]:
```suggestion
			description: ["能让你找到特定的生物群系"]
```

File: config/ftbquests/quests/chapters/basic_create.snbt [hunk 6]
```diff
@@ -258,7 +258,7 @@
 		{
 			x: -3.0d
 			y: 0.0d
-			description: ["Allows you to locate structures"]
+			description: ["能让你定位结构。"]
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/basic_create.snbt]:
```suggestion
			description: ["能让你定位结构"]
```

File: config/ftbquests/quests/chapters/basic_create.snbt [hunk 7]
```diff
@@ -275,7 +275,7 @@
 		{
 			x: 4.5d
 			y: 4.5d
-			description: ["Blank tokens are used to create other tokens which allow you to travel to other dimensions"]
+			description: ["空白通行证可用于制作别的通行证，这些通行证能让你去往别的维度。"]
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/basic_create.snbt]:
```suggestion
			description: ["空白通行证可用于制作别的通行证，这些通行证能让你去往别的维度"]
```

File: config/ftbquests/quests/chapters/basic_create.snbt [hunk 8]
```diff
@@ -302,7 +302,7 @@
 		{
 			x: 0.0d
 			y: -1.5d
-			description: ["Used for most create machines"]
+			description: ["用于制作大部分机械动力的机械。"]
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/basic_create.snbt]:
```suggestion
			description: ["用于制作大部分机械动力的机械"]
```

File: config/ftbquests/quests/chapters/basic_create.snbt [hunk 9]
```diff
@@ -343,7 +343,7 @@
 		{
 			x: -3.0d
 			y: 1.5d
-			description: ["Shift right click to place in modules, Hold right click to perform a area scan"]
+			description: ["Shift右键放入模块，长按右键进行区域扫描。"]
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/basic_create.snbt]:
```suggestion
			description: ["Shift+右键放入模块，长按右键进行区域扫描。"]
```

File: config/ftbquests/quests/chapters/basic_create.snbt [hunk 10]
```diff
@@ -373,10 +373,10 @@
 			]
 		}
 		{
-			title: "Sawmill"
+			title: "锯木机"
 			x: -1.5d
 			y: -1.5d
-			description: ["Cutting up wood, shift right click to saw wood in your inventory"]
+			description: ["切割木材。如果你在配置中允许它，也可以通过潜行从物品栏中直接使用它。"]
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/basic_create.snbt]:
```suggestion
			description: ["切割木材，如果你在配置中允许它，也可以通过潜行从物品栏中直接使用它。"]
```

File: config/ftbquests/quests/chapters/basic_create.snbt [hunk 11]
```diff
@@ -400,7 +400,7 @@
 		{
 			x: -3.0d
 			y: 3.0d
-			description: ["A basic charger to charge items with power"]
+			description: ["一台基础的充电器。"]
```
Comment (AnzhiZhang) [inline config/ftbquests/quests/chapters/basic_create.snbt]:
```suggestion
			description: ["一台基础的充电器"]
```

**Review by AnzhiZhang — COMMENTED** [review]
旋转力是机械动力思索内的定义
image

**Pull request merged**